### PR TITLE
Shell Sort In JAVA

### PR DESCRIPTION
**Pull request by salonilakhotia**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Description

(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

File: Java/sort/ShellSort.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import java.util.*;
+
+public class ShellSort{
+    public static void main(String[] args){
+        int array[] = {30, 25, 77, 16 , 80, 32, 67} ;
```
Comment (atarax665) [inline Java/sort/ShellSort.java]: Take dynamic inputs

File: Java/sort/ShellSort.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import java.util.*;
```
Comment (atarax665) [inline Java/sort/ShellSort.java:1]: Add description of the program.

File: Java/sort/ShellSort.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+public class ShellSort{
+    public static void main(String[] args){
+        int array[] = {30, 25, 77, 16 , 80, 32, 67} ;
+        shellSort(array);
+    }
+
+    public static void shellSort(int array[]){
+        int size = array.length;
```
Comment (atarax665) [inline Java/sort/ShellSort.java]: Include appropriate Documentation (docstrings) for functions. Add comments where needed.

**Review by atarax665 — CHANGES_REQUESTED** [review]

**Comment by salonilakhotia** [comment]
@atarax665 I will do the changes.

File: Java/sort/ShellSort.java [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+            }
+            gap = gap/2;
+        }
+    
+        //printing sorted array
+        System.out.print(" ShellSort : ");
+        for(int i=0;i<size;i++){
+            System.out.print(array[i] + " "); // Printing the sorted array
+        }
+    }
```
Comment (siddharth25pandey) [inline Java/sort/ShellSort.java:44]: Add Sample Input & Output in Comment form

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]

**Comment by siddharth25pandey** [comment]
@salonilakhotia I request you ,from next time onwards Please create separate PR for different issue
Also, add the issue number in PR Template and do the required modifications

**Comment by salonilakhotia** [comment]
@siddharth25pandey I have done the changes !!
I want to ask whether this PR will be accepted for both the questions?

**Comment by siddharth25pandey** [comment]
> @siddharth25pandey I have done the changes !!
> I want to ask whether this PR will be accepted for both the questions?

@salonilakhotia Will be better if you create separate PR for separate Issues. 
From this PR you can only contribute for any one issue.

**Comment by atarax665** [comment]
@salonilakhotia We don't have any updates from you. Make the suggested changes ASAP.

**Review by atarax665 — COMMENTED** [review]
This PR is being closed due to inactivity from contributor's end.

**Pull request closed without merging**